src/orders/components/OrderReturnPage/components/TransactionSubmitCard/TransactionSubmitCard.test.tsx

```diff
@@ -0,0 +1,64 @@
+import { ThemeProvider as LegacyThemeProvider } from "@saleor/macaw-ui";
+import { ThemeProvider } from "@saleor/macaw-ui-next";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import React, { ReactNode } from "react";
+
+import { TransactionSubmitCard } from "./TransactionSubmitCard";
+
+const Wrapper = ({ children }: { children: ReactNode }) => {
+  return (
+    <LegacyThemeProvider>
+      <ThemeProvider>{children}</ThemeProvider>
+    </LegacyThemeProvider>
+  );
+};
+
+jest.mock("react-intl", () => ({
+  useIntl: jest.fn(() => ({
+    formatMessage: jest.fn(x => x.defaultMessage),
+  })),
+  defineMessages: jest.fn(x => x),
+  FormattedMessage: ({ defaultMessage }: { defaultMessage: string }) => (
+    <>{defaultMessage}</>
+  ),
+}));
+
+describe("TransactionSubmitCard", () => {
+  it("submits on click", async () => {
```

Comment:
suggestion: this test check one thing, you have couple checkbox and price field to check also